recipes/default.rb

```diff
@@ -13,12 +13,11 @@
   frequency 3600
 end
 
-# Install docker from docker servers to get latest version supporting nvidia
-# toolkit (at least 19.03)
```

Comment:
I removed it since all version available in Focal for docker are greater than 19.03 so we probably don't need to worry about it anymore. Added a comment about the seccomp issue to document why we need this new version [cf8397f](https://github.com/osrf/osrf_jenkins_agent/pull/27/commits/cf8397fe4fb2531b18dbbf238859fb60378014af)